pull_request_template.md

```diff
@@ -20,4 +20,14 @@
 
 <em>You may erase this after checking them all ;)</em>
 
-- [] CHANGELOG entry added
+- [ ] Added an entry in the `CHANGELOG.md` at the beginning of its due section. [The latest version should comes first](https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first.).
+- [ ] Added the PR number at the entry in the `CHANGELOG.md`.
+
+
+- PR description
+- [ ] Updated the Storybook - *if applicable*.
+- [ ] Added a label according to the PR goal - `Breaking change`, `Enhancement`, `Bug` or `Chore`.
```

Comment:
[Added](https://github.com/vtex-sites/gatsby.store/labels) these 2 missing labels to the repo:
<img width="649" alt="CleanShot 2022-04-26 at 18 18 00@2x" src="https://user-images.githubusercontent.com/381395/165394017-fcca4047-3c92-40b8-8229-6d0225f50447.png">

Reply:
Thanks, @filipewl!